paasta_tools/cli/cmds/status.py

```diff
@@ -840,12 +842,6 @@ def paasta_status(
     :param args: argparse.Namespace obj created from sys.args by cli"""
     soa_dir = args.soa_dir
     system_paasta_config = load_system_paasta_config()
-
-    if 'USE_API_ENDPOINT' in os.environ:
```

Comment:
I think we want to keep this as an option until we have merged @stug 's changes for Marathon and confirmed that everything works on the API as expected.